src/components/Counter/Counter.st.css

```
@@ -43,34 +49,47 @@
     background: color(value(DefaultBackgroundColor));
     border-width: 1px;
     border-style: solid;
-    border-color: color(value(counterColor));
+    border-color: applyOpacity(value(counterColor), 0.6);
 }
 
 .root:error {
     border-color: value(DefaultErrorColor);
 }
 
+.root:hover:not(:disabled):not(:error),
+.root:focus:not(:disabled):not(:error){
+    border-color: value(counterColor);
+}
+
 .root:disabled {
     border-color: color(value(counterDisabledColor));
 }
 
 .btn {
+    -st-extends: TPAIconButton;
     background: none;
     padding: 6px;
     border: none;
     cursor: pointer;
 }
 
-.btn path {
+.root .btn::icon svg path {
     fill: color(value(counterColor));
+    stroke: none;
 }
 
 .btn[disabled] {
     cursor: default;
 }
 
-.btn[disabled] path {
+.root:disabled .btn::icon svg path {
     fill: color(value(counterDisabledColor));
+    stroke: none;
+}
+
+.root .btn[disabled]::icon svg path {
+    fill: color(value(counterDisabledColor)) !important;
+    stroke: none;
 }
```

Similar to the previous comment, you can do:
```suggestion
.btn:disabled {
    -st-mixin: TPAIconButton(
        IconColor color(value(counterDisabledColor))
    );
}
```
But this means the Counter's disabled color is different than the `IconButton`'s disabled color?
Maybe we should ask Johanna or Michal about this.
It seems to me they should be the same color.